packages/react/src/components/Loading/Loading-test.js

```diff
@@ -12,6 +12,17 @@ import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
+describe('Loading - accessibility', () => {
+  it('should have a programatically determinable label', () => {
+    const wrapper = mount(<Loading />);
+    const getLoader = () => wrapper.find('[aria-labelledby]');
```

Comment:
Does grabbing the Loader via this attribute make sense? Is there a better way I should be doing this?

Reply:
Given you seem to be trying to grab the top-level DOM node, would you want [`getDOMNode()`](https://airbnb.io/enzyme/docs/api/ReactWrapper/getDOMNode.html)?

Reply:
It'd work here but it's not great practice considering there could be multiple elements with that property (although I like where your head's at). This would be like querying for `className`, we can assume it's gonna be there but it's an awkward way about it.

You'd want to create a custom label, and query for that text rather than just the property.

One great thing about the RTL `getByLabelText` is it doesn't matter if you use just a  `<label>` or `[aria-labelledby]`, it'll find the correct element. So if we changed our labeling approach down the line, the test wouldn't break.

https://testing-library.com/docs/dom-testing-library/api-queries#bylabeltext